auth-web/src/views/auth/home/HomeView.vue

```diff
@@ -117,6 +117,17 @@
                   <a :href="'mailto:' + $t('techSupportEmail') + '?subject=' + $t('techSupportEmailSubject')">{{ $t('techSupportEmail') }}</a>
                 </li>
               </ul>
+              <p class="mb-5">
+                <strong>{{ $t('labelHoursOfOperation') }}</strong><br>
+                {{ $t('hoursOfOperation') }}
+              </p>
+              <p class="mb-0">For account setup and login support, contact us at:</p>
+              <ul class="contact-info__list mb-5">
+                <li>
+                  <span>{{ $t('labelTollFree') }}</span>
+                  <a :href="`tel:+${$t('maximusSupportTollFree')}`">{{ $t('maximusSupportTollFree') }}</a>
+                </li>
+              </ul>
```

Comment:
And while you're here, this doesn't look right either and maybe needs a prop (this is Business Registry home page, this is not an application):
![image](https://user-images.githubusercontent.com/10002889/199111186-1ae58185-287d-4f88-bfe1-82385ee6101e.png)

^^ Ask UI team about this. This could be a different ticket but you're here already so 🤷

Reply:
Sure. I'll confirm with the UI team.

Reply:
Just confirmed with Yui that "this additional information is only for the footer of this marketing page. No need to propagate to other instances of 'Contact Us'." So this solution should be good.

Reply:
I think we are talking about different pages.

This HomeView page is the Business Registry home page.

Ticket 13765 refers to the BC Registry footer -> https://github.com/bcgov/bcregistry/blob/98fe48225c39a74d5ae59a56aa417e7866bb72b2/layouts/default.vue#L16

I figured you'd need to update the SbcFooter common component (and made this extra info conditional on a prop so that only BC Registry UI shows the extra info, not all UIs that use the SbcFooter).

Reply:
I'll add some links to the ticket to be precise about what needs to be updated.

Reply:
Can you create a different ticket for the issue you are referring to?

Reply:
The subject ticket (13765) is what I am talking about.

This PR is implementing something different than what 13765 is requesting.

Reply:
Ticket 13765 was originally created without the two tasks. The two tasks were added on Friday's grooming session, and I strongly doubt that these two tasks are for this ticket.

Reply:
Yes, I added the 2 tasks to be precise about what this ticket is asking for. This work was always required but was not explained very well unfortunately.

If you are still unclear about what needs to be done and where, please check with Yui and/or Sarah.